src/main/java/org/opentripplanner/routing/edgetype/Timetable.java

```diff
@@ -150,7 +153,12 @@ public TripTimes getNextTrip(State s0, ServiceDay serviceDay, int stopIndex, boo
             int adjustedTime = adjustTimeForTransfer(s0, currentStop, tt.trip, boarding, serviceDay, time);
             if (adjustedTime == -1) continue;
             if (boarding) {
-                int depTime = tt.getDepartureTime(stopIndex);
+                int adjustment = 0;
```

Comment:
`Adjustment` is a very general term. What is this adjusting and what causes this adjustment? What is the flexOffsetScale and the other new parameters? This adds a certain amount of complexity to every departure search, but there are no comments explaining this new code.